flexprep/domain/db_utils.py

```diff
@@ -102,9 +108,9 @@ def update_item_as_processed(self, row_id: int) -> None:
             with self.conn:
                 result = self.conn.execute(
                     """
-                    UPDATE uploaded
```

Comment:
why change the indent?

Reply:
I fixed the indentation to be properly aligned with the other lines of code in the last commit.